Dockerfile

```diff
@@ -7,7 +7,9 @@ FROM bash:${bashver}
 RUN apk add --no-cache parallel && \
     mkdir -p ~/.parallel && touch ~/.parallel/will-cite
 
-RUN ln -s /opt/bats/bin/bats /usr/sbin/bats
-COPY . /opt/bats/
+WORKDIR /opt/bats
```

Comment:
Is this a breaking change? Users expecting an existing workdir will have to update their use of the Docker container?

Reply:
The default workdir for docker is `/`. It could be breaking if people copy do `COPY . to foo` and expect it to be in `/`. Although I think most people either set the workdir before a copy and than use relative paths or they copy it with the absolute path.